scalding-core/src/main/scala/com/twitter/scalding/typed/TypedPipe.scala

```diff
@@ -44,7 +44,20 @@ object TypedPipe extends Serializable {
    */
   def from[T](pipe: Pipe, fields: Fields)(implicit flowDef: FlowDef, mode: Mode, conv: TupleConverter[T]): TypedPipe[T] = {
```

Comment:
Do you think we can pull this method out into an implicit on the typed pipe type in a cascading package/some other package possibly and delurk them from here entirely? its not binary compatible, but if we split scalding-core into some sort of scalding-typed and scalding-core where scalding-core has the cascading dep, we might be able to make it source compatible I think?